docs/examples/make_pymvpa_collection.sh

```diff
@@ -44,37 +32,63 @@ datalad create-handle mnist
         # TODO --doi "10.1109/5.726791"
 )
 
-# Forrest gump dataset, although not yet used in PyMVPA stock materials,
-# analysis scripts use PyMVPA heavily so it is a worthwhile addition
-echo "I: adding forrest_gump dataset"
-datalad install-handle http://psydata.ovgu.de/forrest_gump/.git
-# Disabled for now - it's just a lot. Not needed for testing.
-#(
-#    cd forrest_gump
-#    echo "I: getting forrest_gump content"
-#    datalad get .
-#)
+# add the handle to the collection
+datalad add-handle mnist pymvpa
 
-# new plain collection
-echo "I: creating PyMVPA collection"
-datalad create-collection pymvpa_collection pymvpa # path name
+# Now, we want to add several already existing annexes/handles. There are
+# different possibilities to do this.
+# We want to publish the collection later on, including the handles. So we need
+# to make sure, that the to-be published handles grant access to their actual
+# content.
+#
+# First option:
+# Install that handle locally, get all of its content, that therefore can be
+# pushed on publishing. This approach is necessary only, if we actually want to
+# change the content or the origin is not publicly available or we want to
+# provide an additional copy of that content. This approach could be combined
+# with the second one, to link several sources of the content.
 
-# add all handles to the pymvpa_collection
-echo "I: adding all handles to the PyMVPA collection"
-for h in haxby2001 tutorial_data mnist forrest_gump; do
-    datalad add-handle "$h" pymvpa
-done
+echo "I: adding haxby2001"
+# Haxby 2001 -- the main demo dataset
+datalad install-handle http://data.pymvpa.org/datasets/haxby2001/.git
+(
+    cd haxby2001
+    echo "I: getting haxby2001 content"
+    datalad get .
```

Comment:
re updates:  b) indeed would be the automagic way, for which we would need to register a handle pointing to original repository, not our clone. So actually we would not be able (shouldn't even may be care) to add git special remote
a) I was talking about conscious  decision by us to update the published handle either with our data or may be just to progress forward to the state original repository has it (just "git pull --ff-only" locally and then republish, which might need to update the collection as well if meta-data changes). So that is the use-case I was interested in -- how we push updated collection which should also imply that remote (published) collection repository gets updated with a new commit.